core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -690,6 +690,22 @@ protected double applyMaxSpeed(ReaderWay way, double speed) {
         return speed;
     }
 
+    /**
+     * Checks if a road is access destricted. For example, if we have a case with
+     * access=destination and [encoder_vehicle]=yes, this method returns false
```

Comment:
Is this really true? Would it be possible to add destination to the intendedValues?

But let's clarify if this issue is really worth the effort. Using pushing sections for bike in these cases is ok, but for foot "destination" does not make much sense. And then only bike should be changed.

Reply:
Mhm, not sure if destination is a intended value? In general we want to avoid roads with access=destination?

I think the example was about a road that has for example access=destination and bicycle=yes, so bicycle would be the more specific rule and therefore more important. So we would see this road as a regular road for bicycles, but as access restricted for cars.

> but for foot "destination" does not make much sense

You mean because of incorrectly tagged roads? I think there are not many roads that actually use foot=destination.